docker/editor/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
I think we could ship these files within a docker image and only mount the config as a volume.